packages/kokkos-kernels/common/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 # Adding source directory to the build
 LIST(APPEND KK_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/common/src)
-LIST(APPEND KK_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/common/src/impl)
```

Comment:
L3-4 changes correspond to https://github.com/kokkos/kokkos-kernels/pull/1844 (merged)

Reply:
NOTE: This change to KokkosKernels is only needed to get the new test `TrilinosInstallTests_simpleBuildAgainstTrilinos_by_package_build_tree` to pass.  (And see my comment [above](https://github.com/trilinos/Trilinos/pull/11863#issuecomment-1574011808) about that.)